tasks.py

```diff
@@ -571,16 +571,20 @@ def visit_Call(self, node):
                     # Handle a few different ways that the message can get passed to the deprecation helper
                     # since it's not always a literal string (e.g. it's often a functions .__doc__ attribute)
                     if isinstance(message, ast.Name):
-                        message = self.assignments.get(message.id, "")
+                        assert message  # mypy apparently thinks message can be None here
```

Comment:
Strange this appears here. I thought this was already merged in https://github.com/modal-labs/modal-client/commit/585868c97f901490648bdf92a85e6bc403ac50ea.